website/content/api-docs/secret/aws.mdx

```diff
@@ -41,7 +41,7 @@ files, or IAM/ECS instances.
 At present, this endpoint does not confirm that the provided AWS credentials are
 valid AWS credentials with proper permissions.
 
-### Plugin WIF
+### Plugin WIF <EnterpriseAlert product="vault" inline />
```

Comment:
I'm confused by the enterprise tags here. Is everything in this section Enterprise-only? If the answer is "no", why are we tagging the section header? If the answer is "yes", why are we tagging some parameters but not others?

Reply:
This section is meant to be an enterprise-only alternative to what's described in the IAM section above. I think it would make sense to move the table in this section into a different section? I mostly wanted to indicate that the plugin WIF configuration is enterprise-only in relevant sections and API parameters.

Reply:
If we could group the elements that are enterprise only together and label that, it would help clarify things.

Maybe something like:
```
## Plugin WIF (applies to both)
<intro>

### Basic parameters (applies to both)
<non-enterprise fields>

### Advanced heading (tagged Enterprise to indicate everything in this section is Enterprise)
<enterprise fields>
```